manifest_staging/charts/secrets-store-csi-driver/templates/crds-upgrade-hook.yaml

```diff
@@ -2,7 +2,8 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
   name: {{ template "sscd.fullname" . }}-upgrade-crds
-{{ include "sscd.labels" . | indent 2 }}
+  labels:
+    {{ include "sscd.labels" . | indent 4 }}
```

Comment:
This format needs to be updated in all the other resources in the file and `templates/*` directory

Reply:
> This format needs to be updated in all the other resources in the file and `templates/*` directory

Hum, it already is, isn't it ? @aramase

Reply:
@thib-mary Looking at the `crds-upgrade-hook.yaml` file: https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/acbb36adf6ba8be8d3c8994828dbd7e528cbe43c/manifest_staging/charts/secrets-store-csi-driver/templates/keep-crds-upgrade-hook.yaml, I don't see the change for all other resources.